open_seq2seq/optimizers/optimizers.py

```diff
@@ -75,10 +75,11 @@ def get_regularization_loss(scope=None, name="total_regularization_loss"):
 
 def reduce_gradients(grads_and_vars, on_horovod):
   if on_horovod:
-    from horovod.common import size
```

Comment:
Would be simpler to do a one line code fix
```
from horovod.tensorflow import size
```
and leave the rest of the code unmodified.  `size` is actually a method on `_HorovodBasics` and is set and imported from here https://github.com/uber/horovod/blob/634242ac4b372461259aaee612bb2ca27ef12be4/horovod/tensorflow/mpi_ops.py#L65